src/cpp/lpnamer/problem_modifier/AntaresProblemToXpansionProblemTranslator.cpp

```diff
@@ -18,17 +20,37 @@ AntaresProblemToXpansionProblemTranslator::translateToXpansionProblem(
   problem->init();
   const auto& constant = lps._constant;
   const auto& hebdo = lps._hebdo.at({year, week});
+  problem->_name = hebdo->name;
 
   std::vector<int> tmp(constant->NombreDeVariables, 0);
   std::vector<char> coltypes(constant->NombreDeVariables, 'C');
 
+  auto round10 = []<typename T>(T& collection) {
+    std::ranges::transform(collection, collection.begin(), [](double v) {
+      return round(v * pow(10, 10)) * pow(10, -10);
+    });
+  };
+
+  round10(hebdo->CoutLineaire);
+  round10(hebdo->Xmin);
+  round10(hebdo->Xmax);
+  round10(hebdo->SecondMembre);
+  round10(constant->CoefficientsDeLaMatriceDesContraintes);
+
   problem->add_cols(constant->NombreDeVariables, 0, hebdo->CoutLineaire.data(),
                     tmp.data(), {}, {}, hebdo->Xmin.data(), hebdo->Xmax.data());
+
   problem->add_rows(
       constant->NombreDeContraintes, constant->NombreDeCoefficients,
       convertSignToLEG(hebdo->Sens.data()).data(), hebdo->SecondMembre.data(),
       {}, constant->Mdeb.data(), constant->IndicesColonnes.data(),
       constant->CoefficientsDeLaMatriceDesContraintes.data());
+  for (int i = 0; i < constant->NombreDeVariables; ++i) {
```

Comment:
we have to make sure that the solver does not offer the possibility to rename vars/constr in "one shot" for speed

Reply:
It is indeed possible with Xpress. I don't know about CLP.
Currently we're missing a multisolver interface entry for this. Worst case the CLP implementation would be a loop
I add this to the todo list